lib/constants/constants.js

```diff
@@ -126,17 +126,10 @@ if (PLATFORM_ENV) {
   }
 }
 
-// This value should be ~X.X.X-1
-// where X.X.X maps to version of govuk-frontend used by fb-components-core
-// -1 suffix allows latest published version matching ~X.X.X-*
-const COMPONENTS_VERSION = '~3.1.0-3'
-
 const CONSTANTS = Object.assign({
```

Comment:
What's the benefit of `Object.assign()` here?

This looks to be just a literal, so
```
const CONSTANTS = {
  /* etc */
}
```
Should do

Reply:
i have no idea, this was already here

Reply:
Kill it

Reply:
Current line `151` is also weird

Reply:
how? it's flattening a hash. 

151 is on a different line as it references itself

Reply:
Yeah, that's why it's weird